qualtran/bloqs/basic_gates/swap.py

```diff
@@ -307,7 +306,7 @@ def wire_symbol(self, soq: 'Soquet') -> 'WireSymbol':
             return Circle(filled=True)
 
     def _t_complexity_(self) -> TComplexity:
-        return TComplexity(t=7 * self.bitsize, clifford=10 * self.bitsize)
+        return TComplexity(t=7 * self.bitsize, clifford=14 * self.bitsize)
```

Comment:
Why this change? You decompose into `self.bitsize` `TwoBitCSwap`s, each of which returns `TComplexity(t=7, clifford=10)`

Reply:
`test_t_complexity_cswap` fails if it's set to 10

Reply:
it's because the test utility will go via `to_cirq_circuit` which uses `TwoBitCSwap.as_cirq_op()` to give `cirq.FREDKIN` which has the cliffords we deign not to include

Reply:
changed the offending test